Example/EpoxyExample/ViewControllers/MainViewController.swift

```diff
@@ -118,4 +136,25 @@ final class MainViewController: NavigationController {
     }
   }
 
+  private func makeLayoutGroupsExampleViewController(_ example: LayoutGroupsExample) -> UIViewController {
+    switch example {
+    case .readmeExamples:
+      return LayoutGroupsReadmeExamplesViewController()
+    case .textRowExample:
+      return TextRowExampleViewController()
+    case .colors:
+      return ColorsViewController()
+    case .messages:
+      return MessagesViewController()
+    case .messagesUIStackView:
+      return MessagesUIStackViewViewController()
+    case .todoList:
+      return TodoListViewController()
+    case .entirelyInline:
+      return EntirelyInlineViewController()
+    case .complex:
+      return ComplexDeclarativeViewController()
+    }
```

Comment:
Rather than overriding `title` in each of the example view controllers just suggest setting `viewController.title = example.title` like we do on line 120